python2/doc/genomic.rst

```diff
@@ -361,6 +361,33 @@ one object. The __iadd__ method is overloaded to add elements to the sets:
 
    Instantiation is as in :class:`GenomicArray`, only that ``datatype`` is always ``'O'``.
    
+
+Negative Index
+========
+
+Sometimes a negative index is useful for representing upstream elements. Imagine you would like to pool information about [-500,500) near transcription start site, it would be useful to have a such :class:`GenomicArray`. But unfortunately the current implementation means a such naive approach would raise an error, in
+   
```

Comment:
Thanks for this PR. Please remove the part about naive approaches and the error. We should tell them what to do, not what to NOT do ;-)

Reply:
Feel free to edit. I just find it helpful to record the whole process, just so if people can find this solution if they have a similar error.

Reply:
Ok, I will edit as soon as I find some time.

Reply:
And I feel it justified to contrast two approaches here, even if one of them is not working

Reply:
Yeah I don't feel the same, but thank you, I'll edit and merge.